flow/envs/green_wave_env.py

```diff
@@ -335,6 +335,37 @@ def _split_edge(self, edge):
         else:
             return 0
 
+    def _get_relative_node(self, agent_id, direction):
+        """Yield node number of traffic light agent in a given direction.
+
+        :param agent_id: agent id of the form "center#"
+        :param direction: top, bottom, left, right
+        :return:
+        """
+        agent_id_num = int(agent_id.split("center")[1][0])
```

Comment:
This is generally true in the code base, and I'm sorry to pick on it here when I haven't elsewhere, but is it possible to name these values i.e. [1], [0] is confusing and liable to generate bugs.